src/Razor/src/Microsoft.VisualStudio.LanguageServer.ContainedLanguage/DefaultLSPDocumentManager.cs

```diff
@@ -129,11 +129,18 @@ public override void UntrackDocument(ITextBuffer buffer)
             }
 
             var virtualDocumentAcquired = lspDocument.TryGetVirtualDocument<TVirtualDocument>(out var virtualDocument);
+            if (!virtualDocumentAcquired)
+            {
+                // Unable to locate virtual document of typeof(TVirtualDocument)
+                // Ex. Microsoft.WebTools.Languages.LanguageServer.Delegation.ContainedLanguage.Css.CssVirtualDocument
```

Comment:
I was seeing this. Is this expected?

Reply:
I think this has since been fixed but it's good to have nevertheless

/cc @alexgav @ToddGrun